src/plugins/nginx.go

```diff
@@ -639,3 +641,15 @@ func isFileInDirectoryMap(directoryMap *proto.DirectoryMap, path string) bool {
 	}
 	return false
 }
+
+func isNginxAppProtectEnabled(loadedConfig *config.Config) bool {
```

Comment:
Could you move this function to `src/core/config/types.go` and make it more generic by calling it something like `IsExtensionEnabled`.
Something similar to the `IsFeatureEnabled` function that we have https://github.com/nginx/agent/blob/main/src/core/config/types.go#L40

Reply:
Done